docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,514 @@
-        required: false,
-        description: 'An object representing the zIndex value of the Tooltip.',
+        description: 'Specifies the stacking order of the Tooltip along the z-axis',
+        href: 'Z-index',
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    name="Tooltips"
-    description="General Tooltip usage involves wrapping your target in a Tooltip. Notice how long
-    text is wrapped around underneath."
-    defaultCode={`
-<Tooltip inline text="Logout">
-  <Icon
-    accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use a Tooltip to describe the function of an interactive element, typically an [Icon Button](/IconButton), in as few words as possible."
+        defaultCode={`
+<Tooltip text="Send">
+  <IconButton
+    accessibilityLabel="Send this Pin"
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
   />
+</Tooltip>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Don’t use a Tooltip to restate text already visible on screen."
+        defaultCode={`
+<Tooltip text="Save">
+  <Button color="red" text="Save" inline />
 </Tooltip>
 `}
-  />,
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Tooltips to distinguish related actions with visually similar iconography."
+        defaultCode={`
+<Flex gap={1}>
+  <Tooltip text="Align left">
+    <IconButton
+      accessibilityLabel="Align left"
+      bgColor="white"
+      icon="text-align-left"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align center">
+    <IconButton
+      accessibilityLabel="Align center"
+      bgColor="white"
+      icon="text-align-center"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align right">
+    <IconButton
+      accessibilityLabel="Align right"
+      bgColor="white"
+      icon="text-align-right"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Don’t use a Tooltip to communicate critical information, such as an error or other interaction feedback."
+        defaultCode={`
+<Tooltip text="Pssst! Looks like you've already saved this Pin.">
+  <Button color="red" text="Save" inline />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`Be sure to include an \`accessibilityLabel\` for the screen reader within the child element. While the Tooltip provides a more visible description on screen, its \`text\` property won’t get picked up by the screen reader.
```

Comment:
The Tooltip text does actually get read by a screenreader, so maybe instead: 

"When using a Tooltip with an IconButton, avoid repetitive labelling. The \`accessibilityLabel\` provided to IconButton should describe the intent of the button, and the Tooltip content can expand upon that intention. If the Tooltip content and \`accessibilityLabel\` would contain the same content, do not add an \`accessibilityLabel\` to the IconButton."

Reply:
Also, given these guidelines, it might be good to change this example to something like a warning IconButton, with accessibilityLabel="Errors detected", and then a Tooltip with text="Account email must be verified before you can proceed" and then Another IconButton like the one you have, where the Tooltip is text="Send Pin" and the IconButton does not have an accessibility label

Reply:
DONE